src/index.html

```diff
@@ -11,8 +12,67 @@
       rel="stylesheet"
       href="./styles/index.scss"
     />
+
+    <!--#google Font-->
+    <!--crossorigin-->
+    <link
+      rel="stylesheet"
+      href="./style.css"
+    />
+    <link
+      rel="preconnect"
+      href="https://fonts.googleapis.com"
+    />
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+      crossorigin=" "
```

Comment:
The `crossorigin` attribute is set with an empty value. It should be set to `anonymous` to improve performance when fetching resources from Google Fonts.